cordformation/src/test/resources/net/corda/plugins/DeployThreeNodeCordappWithNetworkConfigWithDocker.gradle

```diff
@@ -0,0 +1,56 @@
+plugins {
+    id 'net.corda.plugins.cordformation'
+}
+
+apply from: 'repositories.gradle'
+apply from: 'postgres.gradle'
+
+dependencies {
+    cordaRuntime "$corda_group:corda:$corda_release_version"
+    cordaRuntime "$corda_group:corda-node-api:$corda_release_version"
+    cordapp "$corda_group:corda-finance-contracts:$corda_release_version"
+    cordapp "$corda_group:corda-finance-workflows:$corda_release_version"
+}
+
+task prepareDockerNodes(type: net.corda.plugins.Dockerform, dependsOn: ['jar']) {
```

Comment:
I think your `Dockerform` task also needs to depend on the `generateInitScripts` task declared in the `postgresql.gradle` plugin. But you only seem to have done this in `DeploySingleNodeWithExternalDBSettingsWithDocker.gradle` :confused:.